src/elements/public/Admin/Log/ErrorEntry.ts

```diff
@@ -0,0 +1,444 @@
+          }
+        }
+      `
+    ];
+  }
+
+  static get properties(): PropertyDeclarations {
+    return {
+      ...super.properties,
+      open: { type: Boolean },
+      nohide: { type: Boolean },
+    };
+  }
+
+  public static get scopedElements(): ScopedElementsMap {
+    return {
+      'iron-icon': customElements.get('iron-icon'),
+      'foxy-i18n': customElements.get('foxy-i18n'),
+      'vaadin-button': ButtonElement,
+      'vaadin-checkbox': CheckboxElement,
+      'foxy-custom-box': Box,
+      'foxy-customer-info': CustomerInfo,
+      'foxy-client-info': ClientInfo,
+      'foxy-transaction-info': TransactionInfo,
+      'foxy-params-viewer': URLSearchParamsViewer,
+    };
+  }
+
+  open: boolean;
+
+  nohide = false;
+
+  constructor() {
+    super();
+    this.open = false;
+  }
+
+  render(): TemplateResult {
+    if (!this.data) {
+      return html`
+        <div class="absolute inset-0 flex items-center justify-center">
+          <foxy-spinner
+              data-testid="spinner"
+              class="p-m bg-base shadow-xs rounded-t-l rounded-b-l"
+              layout="horizontal"
+              >
+          </foxy-spinner>
+        </div>
+      `;
+    } else {
+      return html`
+        <div aria-busy=${this.in('busy')}
+             aria-live='polite'
+             class='${this.nohide? 'nohide': ''} ${ this.data.hide_error ? 'fadeout': ''} flex flex-auto content-center w-full ${this.data.hide_error ? 'text-tertiary': ''}'>
+          <vaadin-checkbox
+            title="Hide this error"
+            class="pt-s"
+            ?disabled=${!this.in('idle')}
+            .value=${this.data._links.self.href}
+            ?checked=${this.data.hide_error}
+            @change=${this.__handleCheckErrorEntry}
+          ></vaadin-checkbox>
+          <article class='m-s text-body'>
+            <header class='border-l-2 p-s relative ${this.open ? 'border-error' : 'border-primary'}'>
+              <vaadin-button @click=${this.__toggleOpen}
+                    theme="icon"
+                    class='text-s absolute right-s top-s rounded-full bg-transparent top-0 m-0 p-0'
+                    aria-label='toggle'
+                    >
+                    ${this.open
+                        ? html`<iron-icon icon='icons:expand-less'></iron-icon>`
+                        : html`<iron-icon icon='icons:expand-more'></iron-icon>`
+                        }
+              </vaadin-button>
+              ${this.data.hide_error ? html`<span class="px-s py-xs text-s font-medium tracking-wide rounded bg-success-10 text-success">Hidden</span>` : ''}
+              <foxy-i18n key='date' options='{"value": "${this.data.date_created}"}' class='text-s ${this.open ? 'text-error' : 'text-primary'}'></foxy-i18n>
+              <foxy-i18n key='time' options='{"value": "${this.data.date_created}"}' class='text-s ${this.open ? 'text-error' : 'text-primary'}'></foxy-i18n>
+              <div>${this.data.error_message}</div>
```

Comment:
**issue:** error message needs to be truncated to one line when collapsed

Reply:
2f6a2c85ece4b3a103b000649e64a0b346810c5e